apps/src/javalab/JavalabEditor.jsx

```diff
@@ -69,37 +81,47 @@ class JavalabEditor extends React.Component {
     };
   };
 
+  renameFileComplete = e => {
+    e.preventDefault();
+    this.props.renameFile(this.state.oldFilename, this.props.filename);
+    this.setState({renameFileActive: false});
+  };
+
+  enableRenameFile() {
```

Comment:
2 nits: 
1. `activateRenameFile` might be a more accurate name?
2. some functions in this file are arrow functions and some aren't. i'm not sure what our team's current preference is (although i generally see us use arrow functions more often in React components), but we should be consistent